nodeadm/internal/containerd/base-runtime-spec.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
out of curiosity, how this been generated? Is there any reference I can read to understand more in case we need to debug sth for this?

Reply:
I derived this from a few sources:
1. The default base OCI spec given by `ctr oci spec`.
2. An actual spec for a running container grabbed with `ctr container info`.
3. The base spec used by Bottlerocket for this same purpose (providing RLIMIT overrides): https://github.com/bottlerocket-os/bottlerocket/blob/b3e476f382dbaf6bb6bea20d99d3bf143662cef6/packages/containerd/containerd-cri-base-json#L4

The spec itself is thoroughly documented here: https://github.com/opencontainers/runtime-spec